server/methods/alexaApiCall.js

```diff
@@ -0,0 +1,24 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { settings } from '../../app/settings';
+
+Meteor.methods({
+	registerAlexaUser(serverurl, servername, token) {
+		const apiUrl = settings.get('Register_Alexa_Enable_Server_Proxy_URL');
+		const userid = Meteor.userId();
```

Comment:
You should check if the userId exists to prevent unauthenticated calls like we do on other methods https://github.com/RocketChat/Rocket.Chat/blob/develop/server/methods/readMessages.js#L13

You should check the type of the parameters as well like here https://github.com/RocketChat/Rocket.Chat/blob/develop/server/methods/readMessages.js#L9

And both checks should happen before the settings get.